llvm/test/CodeGen/ARM/fast-isel-call.ll

```diff
@@ -296,4 +298,13 @@ define void @call_undef_args() {
   ret void
 }
 
+define void @double_softfp_call() ssp {
+entry:
+; ARMEB-SOFTFP-LABEL: double_softfp_call:
+; ARMEB-SOFTFP: vmov r1, r0, {{d[0-9]+}}
```

Comment:
It will depend the constant is that gets loaded, whether this is the correct order of registers or not. It might be worth adding check lines for the constant pool and the load too.